static/src/javascripts/projects/common/modules/identity/sign-in-gate/index.js

```diff
@@ -60,12 +60,23 @@ const show: () => Promise<boolean> = () =>
             `${config.get('page.host')}/${config.get('page.pageId')}`
         );
 
+        // get the view id to attach to component event params
+        let viewId = '';
+        if (
+            window.guardian &&
```

Comment:
it would be so nice if frontend supported new optional chaining syntax...

Reply:
ikr would be so much nicer! 🙄